### ISSUE TITLE
`DEV` The `set_component_color` is being called for `home.bt_notific` before the values for color are received from the blueprint

### ISSUE THREAD

**Issue by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#1528]
The `set_component_color` is being called for `home.bt_notific` before the values for color are received from the blueprint:

```log
[13:15:54][E][script.set_component_color:2610]:   Component:       home.bt_notific
[13:15:54][E][script.set_component_color:2611]:   Foreground size: 0
```

This is not a major issue, but keeping here for tracking as I wanna fix this later.

**Comment by edwardtfn** [comment]
Fixed!

**Comment by edwardtfn** [comment]
Looks like this still happening: https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/1556#discussioncomment-8087542

**Comment by edwardtfn** [comment]
Fixed!

**Comment by edwardtfn** [comment]
We just released a patch with solves this: [v4.2.5 - Celebrating 1000 Stars with Enhanced Functionality and UI Improvements](https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.2.5)
Could you please update your system and let me know if the issue was solved?
